src/query/storages/index/src/range_filter.rs

```diff
@@ -628,6 +628,41 @@ impl<'a> VerifiableExprBuilder<'a> {
                     "Cannot build atom expression by the operator: not like",
                 ))
             }
+            "in" => {
```

Comment:
is supported `not in`?

Reply:
"not in" is not commonly used, we can support it later.